src/swap/saga.ts

```diff
@@ -0,0 +1,109 @@
+import { CeloTx } from '@celo/connect'
+import { TxParamsNormalizer } from '@celo/connect/lib/utils/tx-params-normalizer'
+import { ContractKit } from '@celo/contractkit'
+import { call, put, takeEvery } from 'redux-saga/effects'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import {
+  swapApprove,
+  swapError,
+  swapExecute,
+  swapPriceChange,
+  swapStart,
+  swapSuccess,
+} from 'src/swap/slice'
+import { sendTransaction } from 'src/transactions/send'
+import { newTransactionContext } from 'src/transactions/types'
+import { fetchWithTimeout } from 'src/utils/fetchWithTimeout'
+import Logger from 'src/utils/Logger'
+import { getContractKit } from 'src/web3/contracts'
+import networkConfig from 'src/web3/networkConfig'
+import { getConnectedUnlockedAccount } from 'src/web3/saga'
+import { applyChainIdWorkaround, buildTxo } from 'src/web3/utils'
+
+const TAG = 'swap/saga'
+
+function getPriceDiff(price1: number, price2: number) {
```

Comment:
can we make this function name more descriptive? i don't really understand why we are dividing by 2, but otherwise it is the price difference percentage right?

Reply:
Added a comment and used the remote config value in https://github.com/valora-inc/wallet/pull/2891/commits/debfbea4e50ffa7eb1b7ecc0be6b08e84fba5bea!

Reply:
`getPercentageDifference`?